appinventor/blocklyeditor/src/blocks/logic.js

```diff
@@ -606,7 +606,7 @@ var SwTerms = {
 			"} ORDER BY ?uri";
 		// console.log(query);
 
-    var endpoint = "http://ppr.cs.dal.ca:3010/terms/query";
+    var endpoint = "https://punya-sparql.csail.mit.edu/terms";
```

Comment:
@william-vw Are you sure that dropping the `/query` part of the URL is correct here?

Reply:
I've tested it and it seems work fine.